src/Navigation/Navigation.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './Navigation.css';
+
+function Navigation() {
+  return (
+    <nav className="navigation">
+      <a className="navigation__link" href="#about">About</a>
+      <a className="navigation__link" href="#services">Services</a>
+      <a className="navigation__link" href="#contact">Contact</a>
+    </nav>
+  );
+}
```

Comment:
I'm not sure, but I think it's better to use arrow function
```suggestion
const Navigation = () => (
  <nav className="navigation">
    <a className="navigation__link" href="#about">About</a>
    <a className="navigation__link" href="#services">Services</a>
    <a className="navigation__link" href="#contact">Contact</a>
  </nav>
);
```

Reply:
Ilya is right, we usually use arrow functions. What's more, you can add export right here
```suggestion
export const Navigation = () => (
    <nav className="navigation">
      <a className="navigation__link" href="#about">About</a>
      <a className="navigation__link" href="#services">Services</a>
      <a className="navigation__link" href="#contact">Contact</a>
    </nav>
)
```